parlai/crowdsourcing/tasks/fast_acute/analysis.py

```diff
@@ -648,6 +654,66 @@ def save_results(self, path: str = None):
                 f.write(compiled_text)
 
 
+class MultiRunAcuteAnalyzer(AcuteAnalyzer):
```

Comment:
It's great to have this!!! Btw can `MultiRunAcuteAnalyzer` handle different run_ids with different eval questions? It seems like the win dataframe is calculated per model pair excluding the eval question?

Reply:
Yeah, good point - I'm now checking that only one eval question is used across all results, both within one run and across runs